### ISSUE TITLE
Hey vco!

### ISSUE THREAD

**Issue by maybe7x** [vcokltfre/aocl#1]
I remembered your name, ALL THE STUPID CHARACTERS (OKAY I KNEW THE FIRST 3 BY HEART)! (It is fine to shitpost here right..? right? I'll delete if you say so!)

I'd personally prefer [APL](https://xpqz.github.io/learnapl/intro.html) for AoC though. It seems very apt towards my coding style (;.

There's a bit of an AoC competition going on with my work buddies this year, so it is going to be some very unpleasant nights!

ALSO I CAN EXPLAIN WHY I WAS GONE WITHOUT ANY NOTICE: Changed phones, lost the stupid TOTP (Blame Google that they don't back it up in the clouds! Wait.. maybe that's a good thing.) and then rest is history.

Make Maze send me cat pics! I checked out her GitHub too! (MazeOfEncryption! Definitely didn't take a while to find the last word!) (OKAY MAZE I'M SORRY!)

<sub><sub>Also I should probably make a Discord. *Again*.</sub></sub>

❤️ - syl / 7x. <sub><sub>I should've translated this all to Welsh to confuse passersby but that's too much shitposting, even for.. me.</sub></sub>

**Comment by vcokltfre** [comment]
yeah you should make a discord again lol it's been ages, come hang out in .gg/nextcord
